pkg/reconciler/events/pubsub/pubsub_test.go

```diff
@@ -173,7 +173,7 @@ func TestAllCases(t *testing.T) {
 			),
 		},
 		WantEvents: []string{
-			Eventf(corev1.EventTypeNormal, "Updated", "Updated CloudPubSubSource: \"%s/%s\"", testNS, pubsubName),
+			Eventf(corev1.EventTypeNormal, "CloudPubSubSourceReconciled", `PullSubscription reconciled: "%s/%s"`, testNS, pubsubName),
```

Comment:
use the const... reconciledSuccessReason

Reply:
done